src/lib/orderModification.ts

```diff
@@ -109,3 +109,15 @@ export const cancelOrderAsync = (
     });
   });
 };
+
+export const isNewOrderPriceValid = (order: SubaccountOrder, newPrice: number) => {
```

Comment:
Can we brainstorm a good way to add this to unit tests `i.e. orderModification.spec.ts` considering we need abacus state?